src/navigation/Routes.android.js

```diff
@@ -402,7 +402,7 @@ function AuthNavigator() {
       <AuthStack.Screen
         component={PinAuthenticationScreen}
         name={Routes.PIN_AUTHENTICATION_SCREEN}
-        options={{ ...sheetPreset, gestureEnabled: false }}
+        options={{ ...sheetPreset, gestureEnabled: true }}
```

Comment:
If the user reaches the pin sheet, and doesn't know their pin or wants to back out, they can't and have to quit the app and reopen. This allows them to swipe down to dismiss at least.

Reply:
im worried about how this will effect some of the views currently protected by this flow, we may need to add handling for them

Reply:
Removed 👍